INSTALL.md

```diff
@@ -121,32 +98,16 @@ sudo mysterium_client --data-dir=/var/lib/mysterium-client --config-dir=/etc/mys
 ```
 
 
-## Mysterium VPN node (standalone Linux binaries)
+## Mysterium VPN node and client standalone binaries (.tar.gz)
 ### Download
- * https://github.com/MysteriumNetwork/node/releases/download/{VERSION}/mysterium_server_linux_amd64
- * https://github.com/MysteriumNetwork/node/releases/download/{VERSION}/mysterium_server_linux_armhf
-### Running
+ * https://github.com/MysteriumNetwork/node/releases/download/{VERSION}/mysterium_{client or server}_{OS and ARCH}.tar.gz
+### Extract
 ```bash
-mysterium_server --help
-sudo mysterium_server --config-dir=/etc/mysterium-node --identity=0x123456..
+tar -xvzf <downloadedpackage>.tar.gz
 ```
 
-
-## Mysterium VPN client (standalone Linux binaries)
-### Download
- * https://github.com/MysteriumNetwork/node/releases/download/{VERSION}/mysterium_client_linux_amd64
- * https://github.com/MysteriumNetwork/node/releases/download/{VERSION}/mysterium_client_linux_armhf
 ### Running
 ```bash
-mysterium_client --help
-sudo mysterium_client && mysterium_client --cli
+cd <extracedir>
+sudo ./<mysterium_client or mysterium_server>
```

Comment:
bit overkill, i'd just split into two commands.

Reply:
Splitted into separate sections for client and node